diff --git a/node_modules/@types/async-retry/index.d.ts b/node_modules/@types/async-retry/index.d.ts
index ed730fe..20c0f2c 100755
--- a/node_modules/@types/async-retry/index.d.ts
+++ b/node_modules/@types/async-retry/index.d.ts
@@ -49,7 +49,7 @@ declare namespace AsyncRetry {
      * @param bail A function you can invoke to abort the retrying (bail).
      * @param attempt The attempt number. The absolute first attempt (before any retries) is `1`.
      */
-    type RetryFunction<TRet> = (bail: (e: Error) => void, attempt: number) => TRet | Promise<TRet>;
+    type RetryFunction<TRet> = (bail: (e: Error) => void, attempt: number, rateLimiter: any) => TRet | Promise<TRet>;
 }
 
 export = AsyncRetry;
